panos_upgrade_assurance/firewall_proxy.py

```diff
@@ -1402,3 +1402,61 @@ def get_device_software_version(self):
         self._fw.get_device_version()
         fw_version = self._fw.version.replace("-h", ".")
         return version.parse(fw_version)
+
+    def get_fib(self) -> dict:
+        """
+        Get the information from the forwarding information table (FIB).
+
+        The actual API command run is `show routing fib`.
+
+        # Returns
+
+        dict: Status of the route entries in the FIB
+        ```python showLineNumbers title="Sample output"
```

Comment:
I think you should have a new line before the code block. Did you test this locally on a local pan.dev instance?